src/App.tsx

```diff
@@ -1,204 +1,128 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { todosService, USER_ID } from './api/todos';
+
+import { useEffect, useRef, useState } from 'react';
+
+import './styles/todoapp.scss';
+
+import { Todo } from './types/Todo';
+import { ErrorMessages } from './types/errorMessage';
+
+import { Header } from './components/Header';
+import { ErrorNotification } from './components/ErrorNotification';
+import { TodoList } from './components/Main/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { FilterType } from './types/FilterType';
+
+function getFilteredTodosFromApi(todos: Todo[], filter: FilterType) {
```

Comment:
```suggestion
function getFilteredTodos(todos: Todo[], filter: FilterType) {
```
i don't see api call from this fuction